spotbugs/etc/messages.xml

```diff
@@ -8968,11 +8976,28 @@ Using floating-point variables should not be used as loop counters, as they are
         ]]>
     </Details>
   </BugPattern>
+  <BugPattern type="IAOM_DO_NOT_INCREASE_METHOD_ACCESSIBILITY">
+    <ShortDescription>Accessibility of the overridden or hidden method was increased.</ShortDescription>
+    <LongDescription>Accessibility of the overridden or hidden method {1} was increased from {2} to {3}.</LongDescription>
```

Comment:
Are you sure, that the message is correct? I think the indexes are one off, since the `addClassAndMethod()` uses the `addClass()` and the `addMethod()` functions separately. If you look at the annotationList of a bug instance, you will see, that it has five elements, not four.

Reply:
Thanks, you were right. I removed the unnecessary strings. If my PR seems to be okay now, can you please give me an approve, or maybe start a workflow? Also, thanks again for your hard work reviewing my changes!